exec/plugin-commands-script-runners/src/runRecursive.ts

```diff
@@ -164,3 +161,18 @@ export async function runRecursive (
 
   throwOnCommandFail('pnpm recursive run', result)
 }
+
+function getSpecifiedScripts (scripts: PackageScripts, scriptName: string) {
```

Comment:
why is this function not the same as in `run.ts`?

Reply:
In my understanding, `pnpm` executes `node server.js` in `pnpm run start` even if `start` script is not defined in package.json. I can find the logic to do so in `run.ts` but I cannot in `runRecursive.ts`. This diff is occurred from this diff

Reply:
Perhaps we could add a function parameter for controlling this behavior?

Reply:
No parameter is needed. It is better to write another wrapper for `run.st`, which will include the logic for start.

Reply:
It is also a solution.

Reply:
I fixed the code to wrap `runRecursive` getSpecifiedScripts in `run.tsP